tools/code-owners-parser/CodeOwnersParser/MatchedCodeOwnerEntry.cs

```diff
@@ -1,20 +1,46 @@
-using System;
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.Extensions.FileSystemGlobbing;
```

Comment:
As part of this change I assume you can remove the package reference as well https://github.com/Azure/azure-sdk-tools/pull/5030/files#diff-c80850aeaec1dd2efd7977c09fd3a8393b8482a1ecade588e30a23eab4349190R10?